src/components/Modal/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { modalPropsAtom } from '@/atoms/modalProps.atom'
+import { useRecoilValue } from 'recoil'
+import { Button } from '../Button'
+import * as S from './Modal.styles'
+
+const Modal = () => {
+  const modalProps = useRecoilValue(modalPropsAtom)
+  if (!modalProps.isOpen) return
+  return (
+    <S.Container>
+      <S.ContentCard>
+        <S.ModalBody>{modalProps.message}</S.ModalBody>
+        <S.ModalFooter>
+          <Button
+            variant={'outlined'}
+            onClick={() => {
+              modalProps.onClose()
+            }}
+          >
+            확인
+          </Button>
+        </S.ModalFooter>
+      </S.ContentCard>
+    </S.Container>
+  )
+}
```

Comment:
```suggestion
  const { isOpen, message, onClose } = useRecoilValue(modalPropsAtom)
  
  return (
    {isOpen && (
        <S.Container>
        <S.ContentCard>
          <S.ModalBody>{message}</S.ModalBody>
          <S.ModalFooter>
            <Button
              variant={'outlined'}
              onClick={() => onClose()}
            >
              확인
            </Button>
          </S.ModalFooter>
        </S.ContentCard>
      </S.Container>
    )}
  )
}
```

이렇게 하는 게 좀 더 명시적이고 깔끔한 것 같은데 어떠신가요 ~

Reply:
수정해서 다시 올리겠습니다.!

Reply:
[refactor: 모달이 보이는 상태 관련, 분기 처리 수정](https://github.com/2023-chambit-project/KOOSMOOS/pull/64/commits/dc443acd19ad45857974a4bebbced010b8d73124)
  - 수정 완료했습니다.